packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 58.11.0 - August 17, 2023
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- We made a major upgrade to the version of the `@salesforce/core` library. ([PR #5001](https://github.com/forcedotcom/salesforcedx-vscode/pull/5001))
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- We updated the `SFDX_CONTAINER_MODE` variable to `SF_CONTAINER_MODE` to resolve an issue with running the `SFDX: Authorize an Org` command. ([PR #5020](https://github.com/forcedotcom/salesforcedx-vscode/pull/5020))
+
+- We fixed issues under the hood. ([PR #5021](https://github.com/forcedotcom/salesforcedx-vscode/pull/5021))
```

Comment:
This removes warning message of not a SFDX command and runs 3 out of 7 steps of `SFDX: Create and Set Up project for ISV Debugging` successfully.